compat/maven-model/src/test/java/org/apache/maven/model/v4/ModelXmlTest.java

```diff
@@ -56,19 +56,20 @@ void testXmlRoundtripWithProperties() throws Exception {
 
     @Test
     void testNamespaceInXmlNode() throws XMLStreamException {
-        String xml = "<project xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n"
-                + "         xmlns=\"http://maven.apache.org/POM/4.0.0\"\n"
-                + "         xsi:schemaLocation=\"http://maven.apache.org/POM/4.0.0 http://maven.apache.org/POM/4.0.0\">\n"
-                + "  <build>\n"
-                + "    <plugins>\n"
-                + "      <plugin>\n"
-                + "         <m:configuration xmlns:m=\"http://maven.apache.org/POM/4.0.0\" xmlns=\"http://fabric8.io/fabric8-maven-plugin\">\n"
-                + "             <myConfig>foo</myConfig>\n"
-                + "         </m:configuration>\n"
-                + "      </plugin>\n"
-                + "    </plugins>\n"
-                + "  </build>\n"
-                + "</project>";
+        String xml = """
```

Comment:
IDE suggestion. Can not read this without text block.

Reply:
If you can't read it because it contains line breaks, you have other problems. The IDE suggestion is to make use of the new language feature...